hands-on/README.md

```diff
@@ -29,7 +29,7 @@
 3 directories, 19 files
 ```
 
-## Solutions 
+## Solutions
```

Comment:
I think that we can delete this entire section. Doesn't feel great to be linking to an externally hosted CRG training webpages from 2017.. 😬 

Solutions should all be included within the hands-on docs themselves now.

Reply:
This was a linting fix - I'll leave it in for now but made a note about it for when I review this file in the future.